_search-plugins/star-tree-index.md

```diff
@@ -0,0 +1,190 @@
+
+
+## Example mapping
+
+In the following example, index mappings define the star-tree configuration. This star-tree index precomputes aggregations in the `log` index. The aggregations are calculated using the `size` and `latency` fields for all the combinations of values indexed in the `port` and `status` fields:
+
+```json
+PUT logs
+{
+  "settings": {
+    "index.number_of_shards": 1,
+    "index.number_of_replicas": 0,
+    "index.composite_index": true
+  },
+  "mappings": {
+    "composite": {
+      "request_aggs": {
+        "type": "star_tree",
+        "config": {
+          "ordered_dimensions": [
+            {
+              "name": "status"
+            },
+            {
+              "name": "port"
+            }
+          ],
+          "metrics": [
+            {
+              "name": "size",
+              "stats": [
+                "sum"
+              ]
+            },
+            {
+              "name": "latency",
+              "stats": [
+                "avg"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "properties": {
+      "status": {
+        "type": "integer"
+      },
+      "port": {
+        "type": "integer"
+      },
+      "size": {
+        "type": "integer"
+      },
+      "latency": {
+        "type": "scaled_float",
+        "scaling_factor": 10
+      }
+    }
+  }
+}
+```
+
+For detailed information about star-tree index mappings and parameters, see [Star-tree field type]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/star-tree/).
+
+## Supported queries and aggregations
+
+Star-tree indexes can be used to optimize queries and aggregations. 
+
+### Supported queries
+
+The following queries are supported as of OpenSearch 2.18:
+
+- [Term query](https://opensearch.org/docs/latest/query-dsl/term/term/)
+- [Match all docs query](https://opensearch.org/docs/latest/query-dsl/match-all/)
+
+To use queries with a star-tree index, the query's fields must be present in the `ordered_dimensions` section of the star-tree configuration.
+
+### Supported aggregations
+ 
+The following metric aggregations are supported as of OpenSearch 2.18:
+- [Sum](https://opensearch.org/docs/latest/aggregations/metric/sum/)
+- [Minimum](https://opensearch.org/docs/latest/aggregations/metric/minimum/)
+- [Maximum](https://opensearch.org/docs/latest/aggregations/metric/maximum/)
+- [Value count](https://opensearch.org/docs/latest/aggregations/metric/value-count/)
+- [Average](https://opensearch.org/docs/latest/aggregations/metric/average/)
+
+To use aggregations:
+
+- The fields must be present in the `metrics` section of the star-tree configuration.
+- The metric aggregation type must be part of the `stats` parameter.
+
+### Aggregation example
+
+The following example gets the sum of the `size` field for all error logs with `status=500`, using the [example mapping](#example-mapping):
```

Comment:
Can we reword to something similar : 

The following example gets the sum of all the values in the `size` field - for all error logs with `status=500`, using the [example mapping](#example-mapping):